src/app/static/src/tests/components/adr/selectDataset.test.ts

```diff
@@ -528,6 +544,37 @@ describe("select dataset", () => {
         expect(rendered.find("#loading-dataset").exists()).toBe(false);
         expect(rendered.find(Modal).props("open")).toBe(false);
     });
+    
+    it("renders reset confirmation dialog when changing selected dataset", async () => {
+        let store = getStore({
+            selectedDataset: fakeDataset
+        },
+            {datasets: [{...fakeRawDatasets[0], ...fakeRawDatasets[1], resources: [shape]}]}
+        )
+        const rendered = mount(SelectDataset, {
+            store, stubs: ["tree-select"]
+        });
+        rendered.find("button").trigger("click");
+
+        await Vue.nextTick();
+
+        expect(rendered.find(ResetConfirmation).exists()).toBe(true);
+        expect(rendered.find(ResetConfirmation).props("open")).toBe(true);
+        expect(rendered.find(Modal).props("open")).toBe(false);
+        expect(rendered.findAll("p").length).toBe(2);
+        expectTranslated(rendered.findAll("p").at(0),
+            "Changing the selected dataset will result in changes to all subsequent steps being discarded.",
+            "La modification de l'ensemble de données sélectionné entraînera la suppression des modifications de toutes les étapes suivantes.",
+            store);
+        expectTranslated(rendered.findAll("p").at(1),
+            "These steps will automatically be saved in a version. You will be able to reload this version from the Projects page.",
+            "Ces étapes seront automatiquement sauvegardées dans une version. Vous pourrez recharger cette version depuis la page Projets.",
+            store);
+        const buttons = rendered.findAll("button");
+        expectTranslated(buttons.at(3), "Save version and keep editing",
+            "Sauvegarder la version et continuer à modifier", store);
+        expectTranslated(buttons.at(4), "Cancel editing", "Annuler l'édition", store);
```

Comment:
I probably don't need all of this

Reply:
I think it's fine for testing this text to go into the render confirmation dialog tests (and you have added that test), and for this test just to check that the dialog component is rendered with the corrrect props, e.g. that `open` is set to `true` when expected